pkg/await/awaiters.go

```diff
@@ -141,6 +141,8 @@ var awaiters = map[string]awaitSpec{
 		awaitCreation: untilCoreV1PersistentVolumeClaimBound,
 	},
 	coreV1Pod: {
+		// NOTE: Because we replace the Pod in most situations, we do not require special logic for the
+		// update path.
```

Comment:
Is this to say that there's pretty much no input change to a `Pod` that won't cause a replacement?

Reply:
It's basically saying there is no input change to `Pod` won't result in a replacement, _that also requires an await_.

Reply:
got it, thanks.